src/enclave-agent/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "enclave-agent"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+protobuf = { version = "=2.14.0" }
+bytes = "0.4.11"
+libc = "0.2.58"
+byteorder = "1.3.2"
+log = "0.4.11"
+simple-logging = "2.0.2"
+nix = "0.23.0"
+ttrpc = { git = "https://github.com/haosanzi/ttrpc-rust", features = ["async"] }
+ctrlc = { version = "3.0", features = ["termination"] }
+
+# Async runtime
+tokio = { version = "1.14.0", features = ["full"] }
+async-trait = "0.1.42"
+
+image-rs = { git = "https://github.com/zhiwei-intel-h/image-rs", branch = "dev-occlum-adaption", features = ["unionfs_feature"] }
```

Comment:
Do you know when this will be merged into the `main` branch?
Not being on `main` is not a blocker, but I'd like to avoid depending on moving fork if possible.

Reply:
> Do you know when this will be merged into the `main` branch? Not being on `main` is not a blocker, but I'd like to avoid depending on moving fork if possible.

This personal project is temporary and will be merged into the official image-rs in the near future.